README.rst

```diff
@@ -1,9 +1,7 @@
-=========================================
-anesthetic: nested sampling visualisation
-=========================================
-:anesthetic: nested sampling visualisation
-:Author: Will Handley and Lukas Hergt
-:Version: 2.0.0-beta.12
+===========================================
+anesthetic: nested sampling post-processing
+===========================================
+:Authors: Will Handley and Lukas Hergt
 :Homepage: https://github.com/williamjameshandley/anesthetic
 :Documentation: http://anesthetic.readthedocs.io/
```

Comment:
It will feature in the PyPI badge with the new automatic uploads, but not in the text file. The only way I can think to implement this would be to have a check which ensures the readme version number matches the `__version__` variable. Which would you prefer?

Reply:
I like the idea of having a version number in the readme and testing for a version match. GitHub badges are nice, but I also like having a human readable version number in the readme, e.g. when reading the readme from the terminal.

Reply:
OK, I've now implemented a check that the version in README.rst and anesthetic/_version.py match in the `bin/check_version.py` script